room_assistance/indico_room_assistance/templates/event_request_details.html

```diff
@@ -0,0 +1,20 @@
+{% extends 'events/requests/event_request_details.html' %}
+{% from 'message_box.html' import message_box %}
+
+{% block content %}
+    {% if not event_has_room_attached %}
+         {% call message_box('error') %}
+            {% trans -%}
+                This event does not have a location specified thus you cannot request room assistance
```

Comment:
Link to event settings?

Reply:
How would that help if you have the link on the left side? :P

Reply:
I had it in the `vc_assistance` plugin, I think it was suggested by Thomas. I thought this would be the same.

Reply:
Hm, to me it looks like an unnecessary repetition of something that is already clearly available.

Reply:
Ok... nevermind.

Reply:
Maybe I should add it anyway for the sake of consistency. Thanks anyway for having a look at this!